specification/src/specification/mutations/README.md

```diff
@@ -1,6 +1,6 @@
 # Mutations
 
-The mutation endpoint accepts a mutation request, containing a collection of mutation operations to be performed in the context the data source, and returns a response containing a result for each operation.
+The mutation endpoint accepts a mutation request, containing a collection of mutation operations to be performed transactionally in the context the data source, and returns a response containing a result for each operation.
```

Comment:
Can you please add a subsection below, under `Mutation Operations`, with something like the following

```md
### Multiple Operations

If the `mutations.transactions` capability is enabled, then the caller may provide multiple operations in a single request. Otherwise, the caller must provide exactly one operation.

The intent is that multiple operations ought to be performed together in a single transaction. That is, they should all succeed, or all fail together. If any operation fails, then a single `ErrorResponse` should capture the failure, and none of the operations should effect any changes to the data source.
```

Reply:
done.